plugins/Morpheus/templates/dashboard.twig

```diff
@@ -8,7 +8,7 @@
     <![endif]-->
 {% endblock %}
 
-{% set title %}{{ siteName|raw }}{% if prettyDateLong is defined %} - {{ prettyDateLong }}{% endif %} - {{ 'CoreHome_WebAnalyticsReports'|translate }}{% endset %}
+{% set title %}{{ siteName|raw }} - {{ 'CoreHome_WebAnalyticsReports'|translate }}{% endset %}
```

Comment:
`prettyDateLong` does not seem to be used in any other view templates. Should I remove this from `core/Plugin/Controller.php:L(633)`?

Reply:
:+1: